lazy-pages/src/sys.rs

```diff
@@ -15,39 +35,21 @@ cfg_if! {
     }
 }
 
-#[derive(Debug, derive_more::Display, derive_more::From)]
-pub enum Error {
-    #[display(fmt = "WASM memory begin address is not set")]
-    WasmBeginIsNotSet,
-    #[display(
-        fmt = "Exception is from unknown memory (WASM {:#x} > native page {:x})",
-        wasm_mem_begin,
-        native_page
-    )]
-    UnknownMemory {
-        wasm_mem_begin: usize,
-        native_page: usize,
-    },
-    #[display(
-        fmt = "Page data must contain {} bytes, actually has {}",
-        expected,
-        actual
-    )]
-    InvalidPageSize { expected: usize, actual: u32 },
-    #[display(fmt = "Protection error: {}", _0)]
-    #[from]
-    Protect(region::Error),
-    #[display(fmt = "Lazy page error: {}", _0)]
-    #[from]
-    LazyPage(LazyPageError),
-}
-
 #[derive(Debug)]
 pub struct ExceptionInfo {
     /// Address where fault is occurred
     pub fault_addr: *const (),
 }
 
+/// Returns key which `page` has in storage.
+/// `prefix` is current program prefix in storage.
+fn page_key_in_storage(prefix: &Vec<u8>, page: PageNumber) -> Vec<u8> {
+    let mut key = Vec::with_capacity(prefix.len() + std::mem::size_of::<u32>());
+    key.extend(prefix.clone());
+    key.extend(page.0.to_le_bytes().to_vec());
```

Comment:
```suggestion
    key.extend(page.0.to_le_bytes());
```

Reply:
Done

Reply:
@grishasobol, conversations on the PR __<ins>must</ins>__ be resolved by fixing commit or closed by the author if you discussed it

Reply:
@breathx  absolutely disagree